linear_x: 1.0
linear_y: 0.0
linear_z: 0.0 
angular_x: 0.0
angular_y: 0.0 
angular_z: 0.0
obstacledist: 1.5